cirq-core/cirq/contrib/json_test_data/spec.py

```diff
@@ -0,0 +1,35 @@
+# pylint: disable=wrong-or-nonexistent-copyright-notice
+from __future__ import annotations
+
+import pathlib
+
+import cirq
+from cirq.contrib.json import _class_resolver_dictionary
+from cirq.testing.json import ModuleJsonTestSpec
+
+TestSpec = ModuleJsonTestSpec(
+    name="cirq.contrib",
+    packages=[cirq.contrib],
+    test_data_path=pathlib.Path(__file__).parent,
+    not_yet_serializable=[],
+    should_not_be_serialized=[
+        "QuantumVolumeResult",
+        "SwapPermutationGate",
+        "BayesianNetworkGate",
```

Comment:
The first 3 classes are serializable and should not be listed here.

Please add the corresponding `.repr` and `.json` files instead.

After that you can replace the dictionary filtering below with
```
resolver_cache=_class_resolver_dictionary(),
```

Reply:
Resolved in c9f7cdac02af25ab018181906a0ac925fad23862, the classes were moved to the `tested_elsewhere` list.  Creation of `.repr` and `.json` files is deferred to #7520.